Sum con (#32)
chhoumann/P6: sections/facts.tex

It is worth noting that the transformer used here deviates slightly from the original transformer model proposed by \citet{AttentionIsAllYouNeed}, as we have removed the decoder and implemented the time embedding module presented by \citet{time2vec}.
These are described in detail in section \ref{sec:system architecture}.

> We have developed a web application to show the results of all the trained models in a simple graphical user interface. 

Reviewer: ```suggestion
We have developed a web application to show the predictions from all the trained models in a simple graphical user interface. 
```

The user of the application is able to make temperature predictions by selecting a model and dataset from a dropdown menu. 

In the future, it would be interesting to study the reason for the RNN outperforming the LSTM model.
Based on the \nameref{sec:relatedwork} section, we expected the LSTM model to outperform the RNN model on these larger datasets.
One could run these on more and larger datasets to investigate whether this is still the case. 
In addition, it could also be beneficial to experiment further with hyperparameter tuning for all models.

> In the future it would also be beneficial to update the web application to retrain the models daily with new weather data. It would also be beneficial to update the models to do multivariate forecasting.

Reviewer: ```suggestion
In the future, it would also be beneficial to update the web application to retrain the models daily with new weather data. 
It would also be beneficial to update the models to do multivariate forecasting.
```
